snowplow/iglu-client-embedded/schemas/com.metabase/database/jsonschema/1-0-1

```diff
@@ -1,11 +1,11 @@
 {
-  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-1#",
   "description": "Database connection",
   "self": {
     "vendor": "com.metabase",
     "name": "database",
     "format": "jsonschema",
-    "version": "1-0-0"
+    "version": "1-0-1"
```

Comment:
Wondered about that one too, it should be synced with `metabase.analytics.snowplow`, right?